packages/offix-client/package.json

```diff
@@ -43,12 +43,15 @@
     "apollo-client": "2.6.4",
     "apollo-link": "1.2.13",
     "apollo-link-context": "1.0.19",
+    "apollo-link-error": "^1.1.12",
     "apollo-link-http": "1.5.16",
     "apollo-link-retry": "2.2.15",
     "debug": "4.1.1",
     "idb-localstorage": "0.2.0",
     "offix-cache": "0.9.0",
-    "offix-offline": "0.9.0"
+    "offix-conflicts-client": "0.9.0",
+    "offix-offline": "0.9.0",
+    "offix-scheduler": "0.9.0"
```

Comment:
Note - let's  publish new packages manually using different version and then configure proper permissions.